### DISCUSSION TITLE
PKC matrix beta issue report

### DISCUSSION THREAD

**Discussion by Noustaa** [croneter/PlexKodiConnect#1282] [Q&A] [answered]
Hey @croneter !

Is there a specific place to report issues encountered on PKC matrix beta ? (sync stuck because of time out)

Thank you for the hard work as always ! 
So much appreciated

**Comment by croneter** [comment, marked as answer]
No. Just open another "normal" issue here on Github. Chances are that Kodi is causing the bug, though...
